src/main/kotlin/org/rust/lang/core/psi/impl/RustStubbedItemImpl.kt

```diff
@@ -0,0 +1,26 @@
+package org.rust.lang.core.psi.impl
+
+import com.intellij.lang.ASTNode
+import com.intellij.psi.stubs.IStubElementType
+import com.intellij.psi.util.PsiTreeUtil
+import org.rust.lang.core.psi.RustItem
+import org.rust.lang.core.psi.RustNamedElement
+import org.rust.lang.core.psi.RustOuterAttr
+import org.rust.lang.core.psi.RustVis
+import org.rust.lang.core.stubs.RustItemStub
+
+open class RustStubbedItemImpl : RustStubbedNamedElementImpl<RustItemStub>, RustItem {
+
+    constructor(node: ASTNode) : super(node)
+
+    constructor(stub: RustItemStub, nodeType: IStubElementType<*, *>) : super(stub, nodeType)
+
+    override fun getOuterAttrList(): List<RustOuterAttr> =
+        PsiTreeUtil.getChildrenOfTypeAsList(this, RustOuterAttr::class.java)
```

Comment:
Yeah, the only information currently stored in the stubs is the names of the items. Implementing all of the stubs required for the resolve would be the topic of another PR. 

But considering  `getOuterAttrList` I believe it is not required for the resolve, so we can live it PSI based. 


I should note that this `getOuterAttrList` and `getVis` things are basically copy pasted version of the accessors, generated by Grammar Kit. Inheritance, stubs and grammar kit don't play exceptionally good together